app/views/comments/_comments.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1> <%= translation('comments._comments.comments') %> </h1>
-<% if comments.count() > 0 %>
+<% if @comments.count() > 0 %>
```

Comment:
@billymoroney1 This is where the problem is coming from. A controller is passing the variable as `comments` but the view file expects a `@comments` variable. Maybe we could change this back to `comments`.